src/Pages/MembershipApplication/MembershipPlan.js

```diff
@@ -83,45 +89,76 @@ class MembershipPlan extends Component {
 
   render() {
     return (
-      <Container>
-        <h1>Hi! We're glad you're here.</h1>
-        <div className='membership'>
-          <Row className='membership-plan-row'>
-            {this.state.planType.map((type, ind) => (
-              <div
-                className={
-                  type.plan === this.state.activeId
-                    ? 'card membership-card active-plan'
-                    : 'card membership-card'
-                }
-                id={type.plan}
-                key={ind}
-                onClick={this.cardSelected.bind(this, type.plan)}
+      <div className='background'>
+        <Container id='container-membership-plan'>
+          <h1 className='greet' style={{ color: 'white' }}>Welcome</h1>
+
+          <div className='card' >
+            <div className='membership'>
+              <Container className='top-chunk'>
+                <Row className='benefit-block grid'>
+                  <Col className='col'>
+                    <div className='how-to-join-block'>HOW TO JOIN:
+                      <p>1️⃣ SELECT a plan below</p>
```

Comment:
consider doing `array.map` here, we may need to change the steps someday and using an array is scalable